### PR TITLE
Readme plugin

### PR DESCRIPTION
**Pull request by bennypowers**
considerations:
- do we want to add a `--markdown` cli / config option in the PR, or maybe a subsequent PR?

### CODE REVIEW

File: plugins/readme/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "extends": "../../tsconfig.json" }
```
Comment (thepassle) [inline plugins/readme/tsconfig.json:1]: I dont see a tsconfig in the root? 🤔

File: plugins/readme/test/cem-plugin-readme.test.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    plugins: [readmePlugin({
+      from: dirname(fileURLToPath(import.meta.url)),
+      to: './out/README.md',
+      header: 'README.head.md',
+      footer: 'README.foot.md',
+      private: 'details',
+    })],
+  });
+
+  t.plan(1);
```
Comment (thepassle) [inline plugins/readme/test/cem-plugin-readme.test.js:28]: where does `t` come from?

File: plugins/readme/index.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import chalk from 'chalk';
+import hirestime from 'hirestime';
+import mkdirp from 'mkdirp';
```
Comment (thepassle) [inline plugins/readme/index.js:3]: do we really really _need_ these dependencies? 😅

File: package.json [hunk 4]
```diff
@@ -2,14 +2,33 @@
...
+    "@rollup/plugin-replace": "^2.4.2",
+    "@small-tech/esm-tape-runner": "^1.0.3",
+    "custom-elements-manifest": "^1.0.0",
+    "esbuild": "^0.12.15",
+    "nodemon": "^2.0.4",
+    "npm-run-all": "^4.1.5",
+    "rollup": "^2.52.8",
+    "rollup-plugin-terser": "^7.0.2",
+    "tap-spec": "^5.0.0",
+    "tape": "^5.2.2",
```
Comment (thepassle) [inline package.json:26]: Could we align test runners/frameworks for all the packages? it feels a bit wasteful to have different ways of testing for different package

Comment (bennypowers) [reply package.json:26]: yeah np

File: plugins/readme/index.js (hunk 3 above)
Comment (bennypowers) [reply plugins/readme/index.js:3]: _cmmmmoooon_

File: plugins/readme/test/cem-plugin-readme.test.js (hunk 2 above)
Comment (bennypowers) [reply plugins/readme/test/cem-plugin-readme.test.js]: _waves hands mystically_ it doesn't

File: plugins/readme/tsconfig.json (hunk 1 above)
Comment (bennypowers) [reply plugins/readme/tsconfig.json]: now you don't see it in the package either :D

File: plugins/readme/index.js (hunk 3 above)
Comment (thepassle) [reply plugins/readme/index.js:3]: Its just bloat 😕  if we add dependencies for every little thing, repo is gonna end up like storybook 

whats wrong with `fs.mkdirSync(outdir, { recursive: true });`?

Do we _need_ to log it in _yellow_? Cmooooooon

Comment (bennypowers) [reply plugins/readme/index.js]: *sigh* aright I boring'd it up for you 😉

File: plugins/readme/package.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  "version": "0.1.0",
+  "description": "Generate README.md from custom elements sources",
+  "author": "open-wc",
+  "contributors": [
+    "Benny Powers <web@bennypowers.com>"
+  ],
+  "license": "MIT",
+  "scripts": {
+    "test": "uvu test",
+    "test:watch": "uvu -w test"
```
Comment (thepassle) [inline plugins/readme/package.json:12]: uvu doesnt have a watch mode right? or did they add it?

File: plugins/readme/package.json [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    "url": "https://github.com/open-wc/custom-elements-manifest.git",
+    "directory": "plugins/readme"
+  },
+  "type": "module",
+  "main": "index.js",
+  "keywords": [
+    "custom-elements-manifest",
+    "markdown",
+    "plugin",
+    "copy"
```
Comment (thepassle) [inline plugins/readme/package.json:25]: update keywords

File: package.json [hunk 7]
```diff
@@ -2,14 +2,30 @@
...
   "dependencies": {},
-  "devDependencies": {},
+  "devDependencies": {
+    "@rollup/plugin-commonjs": "^19.0.0",
+    "@rollup/plugin-node-resolve": "^13.0.0",
+    "@rollup/plugin-replace": "^2.4.2",
+    "custom-elements-manifest": "^1.0.0",
+    "esbuild": "^0.12.15",
+    "nodemon": "^2.0.4",
+    "npm-run-all": "^4.1.5",
```
Comment (thepassle) [inline package.json:21]: where is this used?

File: plugins/readme/package.json (hunk 5 above)
Comment (bennypowers) [reply plugins/readme/package.json:12]: I think i got that from a tutorial. doesn't work anyways i'll remove

File: package.json (hunk 7 above)
Comment (bennypowers) [reply package.json:21]: build for to-markdown

**Review by thepassle — APPROVED** [review]

**Pull request merged**